content/Hardware Support/MKR Family/Error-on-MKR-CAN-Shield-silk.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Error on MKR CAN Shield silk
+id: 
+---
+
+An issue has been identified with the silk (the printed text and markings) on some Arduino MKR CAN Shield boards, where the bottom silk is mirrored. To avoid confusion, please use the top silk for reference (the top silk is on the side with the 4-pin CAN terminal). Specifically:
```

Comment:
> where the bottom silk is mirrored

Although this phrase appeared in internal discussions, it may not be the most accurate or clear description of the issue. I think it would be better to describe the issue in simple and direct terms: the CANL and CANH pins were mislabeled on the bottom silk, causing the labels to be swapped.

I would also consider splitting the "solutions" from the first paragraph (which is desciribing the issue), to make sure they're visible.

Two two solutions (as I see it) are:

* Looking at the top silk
* Using the +/- pins as a reference

Reply:
[6451331](https://github.com/arduino/help-center-content/pull/405/commits/64513316f12e0e2ea514ca60a57af2d06e7ba7fa)